packages/assets-controllers/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@metamask/assets-controllers",
+  "version": "0.0.0",
+  "description": "Controllers which manage interactions involving ERC-20, ERC-721, and ERC-1155 tokens (including NFTs)",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/controllers/tree/main/packages/assets-controllers#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/controllers/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/controllers.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "changelog:validate": "../../scripts/validate-changelog.sh",
+    "doc": "../../scripts/doc.sh",
+    "test": "../../scripts/test.sh",
+    "test:watch": "../../scripts/test-watch.sh"
+  },
+  "dependencies": {
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/contracts": "^5.7.0",
+    "@ethersproject/providers": "^5.7.0",
+    "@metamask/base-controller": "workspace:~",
+    "@metamask/contract-metadata": "^1.35.0",
+    "@metamask/controller-utils": "workspace:~",
+    "@metamask/metamask-eth-abis": "3.0.0",
+    "@metamask/network-controller": "workspace:~",
+    "@metamask/preferences-controller": "workspace:~",
+    "@types/uuid": "^8.3.4",
+    "@types/web3": "^1.0.6",
```

Comment:
Hmm, previously this was a dev dependency.

Reply:
Should `@types/*` dependencies ever be in `dependencies` though? We don't ship with these types, so it seems like whoever uses them would have to import them.

Reply:
We do ship types for our own code. If they contain direct references to third-party types, we'd want to ship those as well otherwise consumers would get type errors.

Reply:
Right. That makes sense.

Reply:
There are no direct references to web3 in our types, because it's only used as a private property. So this move to devDependencies makes sense.